modules/tensor_mechanics/src/materials/ADLAROMANCEStressUpdateBase.C

```diff
@@ -89,6 +89,10 @@ ADLAROMANCEStressUpdateBase::validParams()
       "Optional forcing function for the creep strain from the previous timestep. If provided, "
       "the old creep strain will be reset to the function value at the beginning of the "
       "timestep. Used for testing purposes only.");
+  params.addParam<Real>("rom_strain_cutoff",
```

Comment:
This shouldn't be a input parameter, since it is ROM specific, and won't change. I'd consider putting it directly into the inherited classes as another material specific ROM value.

Reply:
Won't change in a given simulation I mean...it'll change between ROM's, rather it'll never change between different simulations using the same ROM...

Reply:
`virtual Real romStrainCutoff() = 0;` ?

Reply:
virtual function added, and override in the inheriting test material added